bin/cron_script.sh

```diff
@@ -50,11 +50,19 @@ elif [ "$HOST_CLOUD" = 'azure' ]; then
   sed -i -e '/image_publisher/,+3d' build-azure.json
 
   # Searching and composing URL of to a custom VHD to use, i.e. the latest kubenow stable image
-  vhd_name=$(az storage blob list --account-name "$AZURE_STORAGE_ACCOUNT" --container-name "$AZURE_CONTAINER_NAME" --query [].name --output tsv | grep "kubenow-$CURRENT_VERSION" | grep '.vhd')
+  vhd_name=$(az storage blob list --account-name "$AZURE_STORAGE_ACCOUNT" --container-name "$AZURE_CONTAINER_NAME" --query [].name --output tsv | grep "kubenow-$CURRENT_VERSION[^-abcr]" | grep '.vhd')
+  echo -e "vhd_name is: $vhd_name"
+
+  # Check whether or not id string is empty
```

Comment:
This was not handled before. It is an unlikely case (mostly related to manual manipulation), but now at least we are interrupting the build.